docs/spec.md

```diff
@@ -208,7 +219,8 @@ by some operations. Tokens are used for imposing execution order on operations
 as described in the [Execution](#execution) section.
```

Comment:
That would be out of scope for this PR, but overall a good idea. I've opened a ticket: #1577. If you have the time, please contribute, since this areas of the spec is in need of some love.